src/styles/components/collectioncard.css

```diff
@@ -1,14 +1,17 @@
 .collectionCard {
-    background-color: #1B3F66;
+    background-color: #004B87;
 }
 
 .collectionTitle {
     font-family: 'Roboto Slab', serif;
-    font-size: 1.25rem;
-    color: white;
+    font-size: 1.65rem;
+    margin-top: 3%;
+    margin-bottom: 5%;
+    color: #E0BB56;
```

Comment:
I think we should keep the title of the collection cards as white. This would maintain consistency across the site.